### PR TITLE
setup repo remote and merge

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for your perusal

File: unicoremc/models.py [hunk 1]
```diff
@@ -69,3 +70,17 @@ def create_repo(self, access_token):
 
     def clone_repo(self):
         Repo.clone_from(self.repo_url, self.repo_path())
+
+    def create_remote(self):
+        repo = Repo(self.repo_path())
+        repo.create_remote('upstream', self.base_repo_url)
+
+    def merge_remote(self):
+        # TODO: Use elasticgit.StorageManager.fast_forward('upstream')
```
Comment (miltontony) [inline unicoremc/models.py]: requires [elasticgit #24](https://github.com/universalcore/elastic-git/pull/24)

File: unicoremc/tests/test_states.py [hunk 2]
```diff
@@ -21,23 +22,42 @@ def setUp(self):
             username='testuser',
             email="test@email.com")
 
+        workdir = os.path.join(settings.CMS_REPO_PATH, 'test-source-repo')
+
+        # NOTE: StorageManager.create_storage() should take a repo path
```
Comment (miltontony) [inline unicoremc/tests/test_states.py:27]: raised in [elasticgit #25](https://github.com/universalcore/elastic-git/issues/25)

Comment (smn) [inline unicoremc/tests/test_states.py:27]: Is https://github.com/universalcore/elastic-git/blob/develop/elasticgit/manager.py#L719 perhaps useful here?

**Comment by smn** [comment]
:+1: but try using `self.addCleanup()` for the various stuff you're doing in `tearDown()`

**Pull request merged**